app/helpers/hyrax/hyrax_helper_behavior.rb

```diff
@@ -0,0 +1,383 @@
+# frozen_string_literal: true
+
+# OVERRIDE: Override the file from Hyrax to modify methods usage
+# rubocop:disable Metrics/ModuleLength
+# rubocop:disable Metrics/AbcSize
+# rubocop:disable Metrics/MethodLength
+module Hyrax
+  # Hyrax Helper Behavior
+  module HyraxHelperBehavior
```

Comment:
We already have access to the file that includes this behavior:
`app/helpers/hyrax_helper.rb`
and more preferably: 
`app/helpers/oregon_digital/hyrax_helper_behavior.rb` <= please make your edits here